pvlib/pvsystem.py

```diff
@@ -2048,30 +2051,40 @@ def snlinverter(v_dc, p_dc, inverter):
 
 def adrinverter(v_dc, p_dc, inverter):
     r'''
-    Converts DC power and voltage to AC power using Sandia's
-    Grid-Connected PV Inverter model.
+    PVL_ADRINVERTER Converts DC power and voltage to AC power
+    using Anton Driesse's Grid-Connected PV Inverter efficiency model
 
     Parameters
     ----------
     v_dc : numeric
-        DC voltages, in volts, which are provided as input to the
-        inverter. Vdc must be >= 0.
+        A scalar or pandas series of DC voltages, in volts, which are provided
+        as input to the inverter. If Vdc and Pdc are vectors, they must be
+        of the same size. v_dc must be >= 0. (V)
 
     p_dc : numeric
-        A scalar or DataFrame of DC powers, in watts, which are provided
-        as input to the inverter. Pdc must be >= 0.
+         A scalar or pandas series of DC powers, in watts, which are provided
+         as input to the inverter. If Vdc and Pdc are vectors, they must be
+         of the same size. p_dc must be >= 0. (W)
 
     inverter : dict-like
-        A dict-like object defining the inverter to be used.
+        A struct defining the inverter to be used, giving the
+        inverter performance parameters according to the model
+        developed by Anton Driesse[1].
         A set of inverter performance parameters are provided with
-        pvlib, or may be generated from the library using retrievesam. 
+        pvlib, or may be generated from the library using retrievesam.
```

Comment:
maybe something like "loaded from the supplied data table using retrieve_sam" would be more accurate.